src/userbase-js/webpack.umd.js

```diff
@@ -0,0 +1,13 @@
+const path = require('path')
+const merge = require('webpack-merge')
+const common = require('./webpack.common.js')
+
+module.exports = merge(common, {
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: `userbase.js`,
```

Comment:
Suffix with package.json [version number](https://github.com/encrypted-dev/userbase/pull/38/files#diff-362197395bb0022cadd86af08cfd7156L21) (e.g. userbase-0.0.6.js)?

Reply:
Is there a specific reason for this? Not opposed but it does seem like a not standard thing to do so just wondering the rationale and if there might be some other way to get the same affect.

Reply:
I think the plan is to store every version of the script in S3 and make each one available to the public like this: https://sdk.userbase.dev/userbase-0.0.6.js

We could manually add the suffix when deploying to S3, or do it as early as possible here. I don't have a strong preference on this

Pinging @dvassallo since he was the one who originally had it with the suffix here and has been handling deploying the script

Reply:
@dvassallo any objection to adding the suffix at release rather than compile time?

Reply:
My inclination is to add it at release time, because that's what I think I've seen done usually. Is there any benefit of updating it at build time?

Reply:
To support that approach for fun, everywhere you see the file you know exactly which version it is. But I'd say the cons of that approach outweigh the pros

So, sounds like this is resolved. Looks good